src/unix/notbsd/mod.rs

```diff
@@ -176,8 +176,44 @@ pub const SIGTRAP: ::c_int = 5;
 pub const PTHREAD_CREATE_JOINABLE: ::c_int = 0;
 pub const PTHREAD_CREATE_DETACHED: ::c_int = 1;
 
-pub const CLOCK_REALTIME: ::c_int = 0;
-pub const CLOCK_MONOTONIC: ::c_int = 1;
+#[repr(i32)]
+/// # Examples
+/// ```
+/// fn main() {
+///     let mut ts = libc::timespec { tv_sec: 0, tv_nsec: 0 };
+///     let rc = unsafe { libc::clock_gettime(libc::CLOCK_REALTIME, &mut ts) };
+///     assert_eq!(0, rc);
+///     let rc = unsafe { libc::clock_gettime(libc::CLOCK_MONOTONIC_COARSE, &mut ts) };
+///     assert_eq!(0, rc);
+/// }
+/// ```
```

Comment:
You can leave out the documentation in this library as these are all just straight bindings to libc, no need to document how to use each and every one